src/pretalx/mail/signals.py

```diff
@@ -8,6 +8,18 @@
 As with all event-plugin signals, the ``sender`` keyword argument will contain the event.
 """
 
+queuedmail_pre_send = EventPluginSignal()
+"""
+This signal is sent out before a ``QueuedMail`` will been sent.
+Receivers may set the ``sent`` timestamp to skip sending via the regular
+email backend but shall not alter any other data of the ``QueuedMail``
+instance.
```

Comment:
We can't really enforce the altering of other data here, so we may as well permit it, I think. Adding something like a custom signature to the email text, or adding/changing attachments, is something I could imagine as a use case.